profiles/management/commands/delete_realistic_users.py

```diff
@@ -0,0 +1,18 @@
+"""
+Deletes a set of realistic users that were added to help us test search functionality
+"""
+from django.core.management import BaseCommand
+from django.contrib.auth.models import User
+from courses.models import Program
+from profiles.management.commands.gen_realistic_users import FAKE_PROGRAM_DESC_PREFIX, FAKE_USER_USERNAME_PREFIX
+
+
+class Command(BaseCommand):
+    """
+    Deletes a set of realistic users and programs/courses that were generated to help us test search
+    """
+    help = "Deletes a set of realistic users and programs/courses that were generated to help us test search"
+
+    def handle(self, *args, **options):
+        User.objects.filter(username__startswith=FAKE_USER_USERNAME_PREFIX).delete()
+        Program.objects.filter(description__startswith=FAKE_PROGRAM_DESC_PREFIX).delete()
```

Comment:
yeah that sounds good to me, just something to make clear that if you want program data and so on this command actually takes care of all of it.